docs/guides/how-to-reset-state.md

```diff
@@ -74,6 +74,70 @@ export const resetAllSlices = () => {
 }
 ```
 
+Resetting bound store using Slices pattern
+
+```ts
```

Comment:
Please align the code style to the rest of code snippets (running Prettier may do this automatically for you, see the task in PR description).

Reply:
Done! 😄 Thank you